webpack/stories/docs/tours.md

```diff
@@ -0,0 +1,45 @@
+# Tour
+
+## React Components
+In order to create a tour of your react component, please add a `<YourComponent>Tour.js` file in the component's library
```

Comment:
```suggestion
In order to create a tour of your React component, please add a `<YourComponent>Tour.js` file in the component's folder.
```

Reply:
Is it better to call the files `YourComponent.Tour.js` instead of `YourComponentTour.js`?  That way it would conform to the convention we use with `.stories.js` and `.test.js` files